lightning/src/util/config.rs

```diff
@@ -237,6 +240,23 @@ pub struct UserConfig {
 	pub peer_channel_config_limits: ChannelHandshakeLimits,
 	/// Channel config which affects behavior during channel lifetime.
 	pub channel_options: ChannelConfig,
+	/// If a channel is not set to be publicly announced, we reject HTLCs which were set to be
+	/// forwarded over the channel if this is set to false. This prevents use from taking on
+	/// HTLC-forwarding risk when we intend to run as a node which is not online reliably.
```

Comment:
Maybe clearer phrasing: If this is set to false, we'll refuse to forward HTLCs over any private channels we create.